NuGet.config

```
@@ -14,6 +14,8 @@
     <add key="dotnet31-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
     <!-- Used for the Rich Navigation indexing task -->
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+	<!-- Used for BenchmarkDotNet prerelease packages -->
+	<add key="benchmark-dotnet-prerelease" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />
```

- is the need for yet-another feed temporary❔
- nit: indent more

---

Maybe, maybe not. Shouldn't matter.
```suggestion
    <add key="benchmark-dotnet-prerelease" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />
```

---

> Shouldn't matter.

Agree it _shouldn't_. However, when AzDO feeds are throttled, the problem gets worse the more feeds we have. Extra feeds can increase restore time as well.